js/bars.js

```diff
@@ -424,6 +427,21 @@ historian.Bars.prototype.renderLabels_ = function() {
       }).prop('outerHTML');  // Convert the div to HTML.
       lines.push(rectHtml + entry.value);
     });
+    if (group.source == historian.historianV2Logs.Sources.SENSORSERVICE_DUMP) {
```

Comment:
Can the two if conditions, here and lines 410-412 be combined ?

Reply:
Yes, I will combine them and insert that string on line 411 to the 2nd place of the list then.